torch/csrc/utils/tensor_new.cpp

```diff
@@ -463,13 +469,17 @@ Tensor tensor_ctor(const Type& type, PyObject* args, PyObject* kwargs) {
   auto r = parser.parse(args, kwargs, parsed_args);
   if (r.idx == 0) {
     bool type_inference = r.isNone(1);
+    // args_requires_grad=true if requires_grad is not set or requires_grad=True
+    // args_requires_grad=false requires_grad is set and requires_grad=false
```

Comment:
err, when you say `requires_grad=false`, you mean that it's set, right? So it would be more accurate to just say "args_requires_grad=false  if explicitly requires_grad=False"